tests/forte/utils/payload_factory_test.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2021 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit test for payload factory.
+"""
+import os
+import unittest
+from forte.utils.payload_factory import (
+    AudioPayloading,
+    ImagePayloading,
+    PayloadFactory,
+)
+from ft.onto.payload_ontology import (
+    AudioPayload,
+    ImagePayload,
+    JpegMeta,
+    AudioMeta,
+)
+from forte.data.data_pack import DataPack
+
+
+class PayloadFactoryTest(unittest.TestCase):
+    """
+    Test PayloadFactory.
+    """
+
+    def setUp(self):
+
+        self.f = PayloadFactory()
+
+    def test_image_payloading(self):
+        datapack = DataPack("image")
+        img_meta = JpegMeta(datapack)
+        img_meta.source_type = "local"
+
+        self.f.register(img_meta)
```

Comment:
The purpose of a `register` is to associate something together. For example, here we should associate the `meta` data info with the function to load the meta data, and the function to serialize the meta data. But if you only provide one value to `register`, it cannot achieve what we want.